src/main/java/graphql/language/Field.java

```diff
@@ -257,13 +257,13 @@ public Builder alias(String alias) {
         }
 
         public Builder arguments(List<Argument> arguments) {
```

Comment:
The next level would be for input methods to TAKE immutablke lists.  eg here we throw the responsibility back onto the caller to give us an immutable list.

Thoughts?

If we did this, then `com.google.collect.ImmutableList` becomes implicit API.

Reply:
We don't need to make it explicit: If you give us a (GraphQL Java) Immutable List it is already optimized inside Guava itself.